package.json

```diff
@@ -43,7 +43,7 @@
     "throttleit": "^1.0.0",
     "unlimited": "^1.1.0",
     "webtorrent": "0.x",
-    "webtorrent-app": "feross/webtorrent-app",
+    "webtorrent-desktop": "git+ssh://git@github.com/feross/webtorrent-desktop.git",
```

Comment:
I had to change this to a github url to get it to work locally. I'm a little concerned about the idea of requiring the desktop application (which runs electron-prebuilt postinstall) as a dependency in www. I guess it's working already but seems a little smelly to me.